tests/test_not_found.py

```diff
@@ -27,15 +29,33 @@ def __assert_404_error(resp, check_body=True):
 @pytest.mark.parametrize('request_path', REQUEST_PATH)
 def test_404_get(nhsd_apim_proxy_url, request_path):
     resp = requests.get(f"{nhsd_apim_proxy_url}{request_path}", headers={
-        "Accept": "*/*"
+        "Accept": "*/*",
+        "Content-Type": "application/vnd.api+json"
```

Comment:
Content-Type not valid on GET as there isn't a body being sent with the request.

Reply:
Removed